File: src/main/java/com/example/yourssu2/Yourssu2Application.java
package com.example.yourssu2;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class Yourssu2Application {

	public static void main(String[] args) {
		SpringApplication.run(Yourssu2Application.class, args);
	}

}

File: src/main/java/com/example/yourssu2/Yourssu2Application.kt
package com.example.yourssu2

import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.boot.autoconfigure.SpringBootApplication
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication

@EnableJpaAuditing
@SpringBootApplication
open class Yourssu2Application{
    companion object Yourssu2Application2 {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(Yourssu2Application::class.java, *args)
        }
    }
}
